presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -432,18 +391,77 @@ node.
 You should ensure that the keytab files have the correct permissions on every
 node after distributing them.
 
+Authorization
+=============
+
+You can enable authorization checks for the :doc:`/connector/hive` by setting
```

Comment:
```suggestion
Enable authorization checks for the :doc:`/connector/hive` by setting
```
Imperative is simpler to read and understand, and clearer to the reader what action they must perform to achieve the goal.